### PR TITLE
Documentation of let-morganey-rest's REST-API

### LINKED ISSUE #25: Describe REST-API
Describe REST calls and format of the json-messages in the README.

### PR DESCRIPTION
**Pull request by keddelzz**
closes #25

### CODE REVIEW

**Review by rexim — APPROVED** [review]
LGTM :+1:

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,11 @@
 
 # Created by https://www.gitignore.io/api/windows,intellij,sbt,scala,java,eclipse
 
+*~
+#*
+*#
```
Comment (ForNeVeR) [inline .gitignore:6]: ~~This line is effectively ignoring everything.~~ I can't see what's going on here. I suggest to completely revert these changes from `.gitignore`.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ForNeVeR) [reply .gitignore:6]: Ah sorry, I'm wrong. Although I still can't get what's going on here, I no more reject the changes because of it.

**Review by ForNeVeR — APPROVED** [review]

**Comment by rexim** [comment]
._.

File: .gitignore (hunk 1 above)
Comment (keddelzz) [reply .gitignore:6]: I'm just ignoring all temporary files from emacs. It creates files starting and ending with `#`.

Comment (ForNeVeR) [reply .gitignore:6]: Okay.

**Comment by keddelzz** [comment]
Thank you very much for the review, guys! 👍

**Pull request merged**